### DISCUSSION TITLE
Solax X1-Boost- G4 sensors

### DISCUSSION THREAD

**Discussion by Joe123-B** [wills106/homeassistant-solax-modbus#765] [Q&A]
Hi all, first of all thank you for the developper of this great integration, I couldn't get the Solax cloud one to work and was happy to see there is a HACS integration that was developped to feed data from the Solax Inverters to HA.
after a successfull set up; My device ( X1-Boost-G4) shows 28 entities ( is that about right?) with 4/5 disabled ones that I enabled manually - what's not showing up correctly is the Total PV Power - giving me a total of 55W when I have an installation of 3KW; the current and Voltages sensors; are showing correctly and I can create a new sensor to have PV power as the product of the two but was wondering if anyone else is having the same issue? I would have thought that PV power and total PV would be a by default sensor that would be one of the most sought after? and therefore wouldn't require any manual calculation/creation of a new sensor to calculate it? am I missing something?  thank you

**Comment by wills106** [comment]
I didn't have your Inverter set correctly.
I'll correct it in the next Beta release.

**Comment by Joe123-B** [comment]
Ah OK, that explains it, thank you very much for your response - was going
mad 😄. Any idea, no pressure, when you foresee it being correct? Just so I
can keep an eye for it.
Much appreciated

**Reply by wills106** [reply]
Hopefully within the next day or two.

**Reply by wills106** [reply]
Just released [2024.02.6b3](https://github.com/wills106/homeassistant-solax-modbus/releases/tag/2024.02.6b3) with hopefully fixed entities.

**Comment by Joe123-B** [comment]
Brilliant, thank you very much.

**Comment by Joe123-B** [comment]
that worked like a charm, thanks a million!  The only entity which is still
showing incorrect data is the Today's Yield ( sensor.solax_today_s_yield)
which is showing  a very high number (  8,388,614.6 kWh ) all the rest is
fine!

On Thu, Feb 22, 2024 at 12:37 PM wills106 ***@***.***> wrote:

> Just released 2024.02.6b3
> <https://github.com/wills106/homeassistant-solax-modbus/releases/tag/2024.02.6b3>
> with hopefully fixed entities.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/wills106/homeassistant-solax-modbus/discussions/765#discussioncomment-8555728>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AMTE62PZ7VOBSRGWLARD3VLYU4UYZAVCNFSM6AAAAABDT4LIHGVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DKNJVG4ZDQ>
> .
> You are receiving this because you authored the thread.Message ID:
> <wills106/homeassistant-solax-modbus/repo-discussions/765/comments/8555728
> @github.com>
>


-- 
Youssef Bouanani
0033664637768

**Reply by wills106** [reply]
Just released [2024.02.6b4](https://github.com/wills106/homeassistant-solax-modbus/releases/tag/2024.02.6b4) I was reading "Today's Yield" as a 32bit register by mistake.

**Comment by Joe123-B** [comment]
Excellent thank you, all good now.

**Comment by fatty1811** [comment]
can you run this unit without connecting mains so its just working off solar input
